### DISCUSSION TITLE
计算机学院论文封面

### DISCUSSION THREAD

**Discussion by Yudezhi** [nju-lug/NJUThesis#147] [Ideas]
学硕和专硕封面略有不同
[硕士学位毕业论文封面.zip](https://github.com/nju-lug/NJUThesis/files/8696800/default.zip)

**Comment by atxy-blip** [comment]
https://github.com/nju-lug/NJUThesis/commit/5ed0e46b08b39afdd5a96b05ba05ab02bd651ac5 更新了标签文字压缩的功能。

对于专硕封面的字样，在导言区加
```TeX
\njusetname{major}{专业学位类别(领域)}
```
![image](https://user-images.githubusercontent.com/63113546/168517363-2e87fc6c-0d54-4fc8-b773-26cf8b0452db.png)

**Reply by Yudezhi** [reply]
6呀！多谢~

**Reply by Yudezhi** [reply]
紫色的logo
[nju-emblem2.pdf](https://github.com/nju-lug/NJUThesis/files/8802022/nju-emblem2.pdf)
[nju-name2.pdf](https://github.com/nju-lug/NJUThesis/files/8802023/nju-name2.pdf)
